From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: aq17 <aqiu@pulumi.com>
Date: Thu, 25 May 2023 10:33:38 -0700
Subject: [PATCH 2/9] Shared-features


diff --git a/internal/features/four_point_oh.go b/internal/features/four_point_oh.go
index 4387a9c18f..4013a59e46 100644
--- a/internal/features/four_point_oh.go
+++ b/internal/features/four_point_oh.go
@@ -44,3 +44,9 @@ func FourPointOh() bool {
 func FourPointOhBeta() bool {
 	return FourPointOh() || false
 }
+
+// FORK: Small function to help us not make a breaking change in the Pulumi
+// provider where all Enums have been changed to case sensitive
+func CaseInsensitive() bool {
+	return true
+}
diff --git a/internal/provider/features.go b/internal/provider/features.go
index c1df7fe4b1..f989086226 100644
--- a/internal/provider/features.go
+++ b/internal/provider/features.go
@@ -340,7 +340,7 @@ func schemaFeatures(supportLegacyTestSuite bool) *pluginsdk.Schema {
 
 	return &pluginsdk.Schema{
 		Type:     pluginsdk.TypeList,
-		Required: true,
+		Optional: true,
 		MaxItems: 1,
 		MinItems: 1,
 		Elem: &pluginsdk.Resource{
