From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ramon Quitales <ramon@pulumi.com>
Date: Wed, 9 Aug 2023 13:05:13 -1000
Subject: [PATCH] Remove defaults backup retention policies

Fixes #1318

diff --git a/internal/services/recoveryservices/backup_policy_vm_resource.go b/internal/services/recoveryservices/backup_policy_vm_resource.go
index ecf0cba3fe..9422a2d2eb 100644
--- a/internal/services/recoveryservices/backup_policy_vm_resource.go
+++ b/internal/services/recoveryservices/backup_policy_vm_resource.go
@@ -1339,10 +1339,11 @@ func resourceBackupProtectionPolicyVMSchema() map[string]*pluginsdk.Schema {
 						},
 					},
 
+					// FORK: Removed default value due to pulumi-azure#1318
+					// These modifications can be removed once pulumi-terraform-bridge#577 is fixed.
 					"include_last_days": {
 						Type:     pluginsdk.TypeBool,
 						Optional: true,
-						Default:  false,
 						ConflictsWith: []string{
 							"retention_monthly.0.weeks",
 							"retention_monthly.0.weekdays",
@@ -1453,10 +1454,11 @@ func resourceBackupProtectionPolicyVMSchema() map[string]*pluginsdk.Schema {
 						},
 					},
 
+					// FORK: Removed default value due to pulumi-azure#1318
+					// These modifications can be removed once pulumi-terraform-bridge#577 is fixed.
 					"include_last_days": {
 						Type:     pluginsdk.TypeBool,
 						Optional: true,
-						Default:  false,
 						ConflictsWith: []string{
 							"retention_yearly.0.weeks",
 							"retention_yearly.0.weekdays",
