From 58d10cbf88263c5267806040ee338ae6523d8d3b Mon Sep 17 00:00:00 2001
From: aq17 <aqiu@pulumi.com>
Date: Thu, 25 May 2023 10:33:38 -0700
Subject: [PATCH 4/9] fixup-Shared-features

---
 internal/tf/suppress/deprecated_soon.go | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/internal/tf/suppress/deprecated_soon.go b/internal/tf/suppress/deprecated_soon.go
index 926f57f880..815f6e3006 100644
--- a/internal/tf/suppress/deprecated_soon.go
+++ b/internal/tf/suppress/deprecated_soon.go
@@ -17,3 +17,9 @@ func CaseDifference(_, old, new string, _ *schema.ResourceData) bool {
 	// fields should be case-sensitive, normalize the Azure Resource ID in the Read if required
 	return strings.EqualFold(old, new)
 }
+
+// CaseDifferenceV2Only only suppress case difference for v2.0.
+func CaseDifferenceV2Only(_, old, new string, _ *schema.ResourceData) bool {
+	// FORK: Force attributes to maintain case-insensitivity to avoid breaking changes in Pulumi programs.
+	return strings.EqualFold(old, new)
+}
-- 
2.41.0

