From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: aq17 <aqiu@pulumi.com>
Date: Thu, 25 May 2023 10:33:39 -0700
Subject: [PATCH 8/9] expose-provider


diff --git a/shim/shim.go b/shim/shim.go
new file mode 100644
index 0000000000..cd6b26156c
--- /dev/null
+++ b/shim/shim.go
@@ -0,0 +1,10 @@
+package shim
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/provider"
+)
+
+func NewProvider() *schema.Provider {
+	return provider.AzureProvider()
+}
